app/Svg/Generator.hs

```diff
@@ -229,7 +229,7 @@ textToSVG styled type_ xPos' text =
                       then xPos
                       else xPos')
             ! A.y (stringValue $ show yPos)
-            ! A.transform (stringValue . show . formatTransform $ textTransform text)
+            ! A.transform (stringValue . show $ textTransform text)
```

Comment:
This code as is doesn't really make sense, as `textTransform text` will return a (string representation of a) list, which isn't a valid form for the `transform` attribute.

Please investigate further about why applying this transformation seems to give an incorrect result.